bank-account: added instructions for function implementation (#2260)
exercism/go: exercises/practice/bank-account/.docs/instructions.append.md

# Instructions append

### 1. Open an account

An account can be opened. On opening, its initial balance should not be negative. In case of the negative balance, `nil` is returned.

### 2. Close an account

> An account can be closed. Closed account has balance = 0. But on closing, an actual balance should be returned - `(balance,true)`. Other close requests should finish returning `(0,false)`.

Reviewer: ```suggestion
An account can be closed. When closing an account, the `Close` method must return the balance the account has and a boolean `true` indicating the account was closed successfully. Closing an account does not succeed if the account is already closed. When an account is closed, its balance must be set to `0`.
```

> ### 3. Get account balance 

Reviewer: ```suggestion
## Getting the account balance 
```

Account balance is accessible. If an account is closed, `(0,false)` is returned, otherwise - `(balance,true)`

> ### 4. Deposit/withdraw an account

Reviewer: ```suggestion
## Deposits / Withrawals
```
